src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html

```diff
@@ -142,6 +143,27 @@
       {{name}}: ${{price}}
     </template>
   </amp-list>
+
+  <!-- E-Commerce websites tipically use carousel layout to display personalized content.
+  One way to address this is to have an `amp-list` as the src of `amp-carousel` and the amp-list endpoint
+  needs to return a single entry in `items`, like this:
```

Comment:
Why does it need to return a single entry items?

Reply:
Find the explanation in [https://github.com/ampproject/amphtml/issues/4405#issuecomment-318204399](https://github.com/ampproject/amphtml/issues/4405#issuecomment-318204399)